egs2/reazonspeech/asr1/README.md

```diff
@@ -0,0 +1,22 @@
+# Corpus
+
+**ReazonSpeech** is an open Japanese corpus harvested from terrestrial
+TV programs. It contains more than 10000h hours of Japanese speech,
+sampled at 16kHz.
+
+The dataset is available on Hugging Face. For more details, please visit:
+
+* Dataset: https://huggingface.co/datasets/reazon-research/reazonspeech
+* Paper: https://research.reazon.jp/_static/reazonspeech_nlp2023.pdf
+
+# Results
+
+## Environment
+
+- Python version: `3.8.16`
+- ESPnet version: `espnet 202301`
+- PyTorch version: `pytorch 1.12.1`
+
+## Pretrained model
+
+https://huggingface.co/reazon-research/reazonspeech-espnet-next
```

Comment:
We has a auto-generated results file located in `exp/<your_model>/RESULTS.md` could you please use that to add the results at here?

Reply:
> We has a auto-generated results file located

Sorry, next time we train ReazonSpeech model, we will make sure to
salvage RESULTS.md from training environment.

We periodically train a new model, so it won't be long before we can
follow it up and submit a PR that add the score table.